rootfs/api/models/release.py

```diff
@@ -218,6 +219,9 @@ def rollback(self, user, version=None):
                 raise DeisException('Cannot roll back to initial release.')
 
             prev = self.app.release_set.get(version=version)
+            if prev.failed:
+                raise DeisException('Cannot roll back to failed release.')
```

Comment:
yes it can fail for reasons not related to the build/config but we don't know why the deploy failed so isn't it better to not allow to rollback to a failed release.